packages/console/src/components/TenantEnvTag/index.module.scss

```diff
@@ -1,15 +1,11 @@
 @use '@/scss/underscore' as _;
 
 .tag {
-  display: flex;
-  align-items: center;
+  display: inline-block;
+  vertical-align: middle;
```

Comment:
nit: why the flex box doesn't work?

Reply:
Since the tag's width should be change according to its content. so it should an `inline-block`, or its background will fullfill the whole container.